### ISSUE TITLE
Feature request: app launcher

### ISSUE THREAD

**Issue by thhck** [Liquid-Surf/easy-penny#7]
based on the container type, or maybe from a manifest file, suggest solid app to open a given container with. 

I feel I have already seen solid app launcher somewhere.. maybe a project from @michielbdejong ?

**Comment by Vinnl** [comment]
> I feel I have already seen solid app launcher somewhere.. maybe a project from @michielbdejong ?

I think there have been multiple, but here's one: https://github.com/inrupt/launcher-exploration

**Comment by thhck** [comment]
> I think there have been multiple, but here's one: https://github.com/inrupt/launcher-exploration

Seems that you contributed to this project ;)
Would you like this feature to be merge to main penny ? 

Actually I was about to send you a message to invite you to check the feature request of this repo. I would like to know if you are interested in merging some of them to upstream.

**Comment by Vinnl** [comment]
Yep, but it was a short exploration and Solid looked different back then, so I'm not sure how much of that still transfers :)

I think it would be a bit out of scope for upstream Penny, but I'll keep an eye on what you're doing, maybe I'll change my mind :)
